publisher/product/ontology/build-theallfile.sh

```diff
@@ -31,8 +31,8 @@ function ontologyCreateTheAllTtlFile() {
     python3 ${SCRIPT_DIR}/lib/trigify.py \
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
-      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \
+      --top="https://spec.edmcouncil.org/${ONTPUB_FAMILY^^}/ontology/Metadata${ONTPUB_FAMILY^^}/" \
```

Comment:
Should be:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`